pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -1791,7 +1795,7 @@ func TestComposePeerAuthentication(t *testing.T) {
 			},
 		},
 		{
-			name: "ignore non-empty selector in root namespace",
+			name: "return non-empty selector in root namespace with default",
```

Comment:
hmm, the name is a bit confusing, I think you can keep the old name, it's still to ignore the non-empty selector but just return permissive instead of nil.

Reply:
Thanks! Changed back to the original naming and rebased to master.